base_import_async/models/base_import_import.py

```diff
@@ -161,13 +164,15 @@ def _split_file(self, model_name, translated_model_name,
             attachment = self._create_csv_attachment(
                 fields, data[row_from:row_to + 1], options,
                 file_name=root + '-' + chunk + ext)
-            delayed_job = self.with_delay(
+            delayed_job = self.with_context(
+                job_batch=batch).with_delay(
                 description=description, priority=priority)._import_one_chunk(
                     model_name=model_name,
                     attachment=attachment,
                     options=options)
             self._link_attachment_to_job(delayed_job, attachment)
             priority += 1
+        batch.enqueue()
```

Comment:
This line will be to remove since https://github.com/OCA/queue/pull/151/commits/8e2dcacda8c04bd18c17a62d3351514463f40bc1

Reply:
no, it has to stay now :)